src/protocolsupport/protocol/typeremapper/itemstack/toclient/MonsterEggToPEIdSpecificRemapper.java

```diff
@@ -0,0 +1,28 @@
+package protocolsupport.protocol.typeremapper.itemstack.toclient;
+
+import org.bukkit.entity.EntityType;
+import protocolsupport.api.ProtocolVersion;
+import protocolsupport.protocol.typeremapper.itemstack.ItemStackSpecificRemapper;
+import protocolsupport.protocol.typeremapper.legacy.LegacyMonsterEgg;
+import protocolsupport.protocol.typeremapper.pe.PEDataValues;
+import protocolsupport.protocol.utils.minecraftdata.MinecraftData;
+import protocolsupport.protocol.utils.types.NetworkEntityType;
+import protocolsupport.zplatform.itemstack.ItemStackWrapper;
+import protocolsupport.zplatform.itemstack.NBTTagCompoundWrapper;
+
+public class MonsterEggToPEIdSpecificRemapper implements ItemStackSpecificRemapper {
+
+	@Override
+	public ItemStackWrapper remap(ProtocolVersion version, String locale, ItemStackWrapper itemstack) {
+		NBTTagCompoundWrapper tag = itemstack.getTag();
+		if (tag.isNull()) {
+			return itemstack;
+		}
+		String id = MinecraftData.removeNamespacePrefix(tag.getCompound("EntityTag").getString("id"));
```

Comment:
This may fail if for whatever reason egg didn't have EntityTag

Reply:
@Shevchik wouldn't the same thing happen for pre-1.8 clients? After all, the code for MonsterEggToPEIdSpecificRemapper is exactly the same as the MonsterEggToLegacyIdSpecificRemapper, except the code to transform to PE ID of course.

But if you want it, I could add the tag check.

Reply:
In your case the removeNamespacePrefix will fail, pc monster egg doesn't have it